From 2c42ef5b91f95d03d9689021e9e1e289ff83e1ab Mon Sep 17 00:00:00 2001
From: Baptiste Assmann <bedis9@gmail.com>
Date: Wed, 9 Oct 2013 21:57:02 +0200
Subject: [PATCH 082/179] DOC: missing http-send-name-header keyword in keyword
 table

This one was in the doc but not in the keyword matrix.
---
 doc/configuration.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/doc/configuration.txt b/doc/configuration.txt
index 7aff1e6..ba8057f 100644
--- a/doc/configuration.txt
+++ b/doc/configuration.txt
@@ -1143,6 +1143,7 @@ http-check expect                         -          -         X         X
 http-check send-state                     X          -         X         X
 http-request                              -          X         X         X
 http-response                             -          X         X         X
+http-send-name-header                     -          -         X         X
 id                                        -          X         X         X
 ignore-persist                            -          X         X         X
 log                                  (*)  X          X         X         X
-- 
1.8.1.5

